internal/trigger/server_test.go

```diff
@@ -31,8 +31,11 @@ func TestServerApi(t *testing.T) {
 		ctrl := gomock.NewController(t)
 		defer ctrl.Finish()
 		w := NewMockWorker(ctrl)
-		s := NewTriggerServer(Config{}).(*server)
-		s.worker = w
+		s := &server{
```

Comment:
Why change this?

Reply:
the forced merge of Wenfeng's pr led to the failure of trigger ci, fix here